mono/tests/unhandled-exception-9.cs

```diff
@@ -0,0 +1,16 @@
+using System;
+using System.Diagnostics;
+using System.Threading;
+using System.Threading.Tasks;
+
+class Driver
+{
+	/* expected exit code: 255 */
```

Comment:
If the expected exit code is 255, why is it in the `TESTS_UNHANDLED_EXCEPTION_1_SRC` with expected exit code of 1?